packages/flutter/lib/src/material/checkbox_theme.dart

```diff
@@ -0,0 +1,241 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'material_state.dart';
+import 'theme.dart';
+import 'theme_data.dart';
+
+/// Defines default property values for descendant [Checkbox] widgets.
+///
+/// Descendant widgets obtain the current [CheckboxThemeData] object using
+/// `CheckboxTheme.of(context)`. Instances of [CheckboxThemeData] can be
+/// customized with [CheckboxThemeData.copyWith].
+///
+/// Typically a [CheckboxThemeData] is specified as part of the overall [Theme]
+/// with [ThemeData.checkboxTheme].
+///
+/// All [CheckboxThemeData] properties are `null` by default. When null, the
+/// [Checkbox] will use the values from [ThemeData] if they exist, otherwise it
+/// will provide its own defaults based on the overall [Theme]'s colorScheme.
+/// See the individual [Checkbox] properties for details.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+@immutable
+class CheckboxThemeData with Diagnosticable {
+  /// Creates a theme that can be used for [ThemeData.checkboxTheme].
+  const CheckboxThemeData({
+    this.fillColor,
+    this.mouseCursor,
+    this.checkColor,
+    this.materialTapTargetSize,
+    this.visualDensity,
+    this.focusColor,
+    this.hoverColor,
+    this.splashRadius,
+  });
+
+  /// {@macro flutter.material.checkbox.fillColor}
+  ///
+  /// If specified, overrides the default value of [Checkbox.fillColor].
+  final MaterialStateProperty<Color?>? fillColor;
```

Comment:
Same as for `Radio`, should `activeColor` be themable?

Reply:
Same answer as above.